pkg/scheduler/core/generic_scheduler.go

```diff
@@ -19,7 +19,7 @@ import (
 
 // ScheduleAlgorithm is the interface that should be implemented to schedule a resource to the target clusters.
 type ScheduleAlgorithm interface {
-	Schedule(context.Context, *policyv1alpha1.Placement, *workv1alpha2.ResourceBindingSpec) (scheduleResult ScheduleResult, err error)
+	Schedule(context.Context, *policyv1alpha1.Placement, *workv1alpha2.ResourceBindingSpec, bool) (scheduleResult ScheduleResult, err error)
```

Comment:
I think this parameter could be wrapped as a new struct. Like 
```
Schedule(context.Context, *policyv1alpha1.Placement, *workv1alpha2.ResourceBindingSpec, *ScheduleAlgorithmOption) (scheduleResult ScheduleResult, err error)
```
And `enablePropagateEmptyWorkLoad` could be a member field of `ScheduleAlgorithmOption` for further extension.